### PR TITLE
Revise cancel archive

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8d24270fa9/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-64303ca591/how-to/).

**Comment by WilliamLoy** [comment]
Please redirect the old link for this article. Thank you.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b834fe3476/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f01defd4ea/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6b7616f298/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b7437f646a/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f46435d712/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-480de3d76b/how-to/)

**Comment by WilliamLoy** [comment]
Please notify me before and receive reply from me before merging. Thanks!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7f8c0ccd80/how-to/)

**Comment by stephamon** [comment]
Hey @DonNateR here are my suggestions. let me know if you have any questions:

[Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.docx](https://github.com/rackerlabs/rackspace-how-to/files/1528727/Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.docx)
[Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1528728/Export.an.archive.before.cancelling.a.Cloud.Office.account_SF.pdf)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d8512aec42/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8accc396b7/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-953166c1ed/how-to/). :bow:

File: content/rackspace-email-archiving/export-an-archive-when-cancelling-your-account.md [hunk 1]
```diff
@@ -19,22 +19,39 @@ aside an appropriate amount of time for completion.
...
-- **Difficulty:** Challenging 
-- **Time needed:** Dependent on amount of data being exported
-- **Tools required:**  Cloud Office Control Panel access and Archive Manager access
 
 For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
 
-### Before you export an archive
+### Export options avaliable 
+
+When cancelling services with Cloud Office the following options are available for preserving your archive after account cancellation.
```
Comment (cyrichardson) [inline content/rackspace-email-archiving/export-an-archive-when-cancelling-your-account.md]: Cloud Office, the following

File: content/rackspace-email-archiving/export-an-archive-when-cancelling-your-account.md [hunk 2]
```diff
@@ -19,22 +19,39 @@ aside an appropriate amount of time for completion.
...
+
+|Archive options |Time Required | Cost | Technical Difficulty |
+|---|---|---|---|
+| Transfer to Sonian | No time required | No transfer cost | No technical work |
+| Self export the archive | Days or weeks | No cost | Difficult to complete |
+| Bulk archive export request | Weeks | Priced per GB exported | Moderate technical work after receiving the export |
+
+### Migrate your archive using Sonian (recommended)
+
+Archiving services are provided by Rackspace’s partner, Sonian Inc. Their cloud email archiving solution is compatible with all of leading email service providers such as Office 365, Gmail, and Exchange.
```
Comment (cyrichardson) [inline content/rackspace-email-archiving/export-an-archive-when-cancelling-your-account.md]: providers, such as

File: content/rackspace-email-archiving/export-an-archive-when-cancelling-your-account.md [hunk 3]
```diff
@@ -19,22 +19,39 @@ aside an appropriate amount of time for completion.
...
+
+This option requires no additional work. You maintain uninterrupted access to all archive data and attachments.
+
+To move archiving support to Sonian directly, simply click [Transfer to Sonain](https://www.sonian.com/rackspacearchive), complete the short form on the page that follows, and Sonian will contact you to ensure a seamless transition of your existing archive to your new mail service.
+
+### Export you own archive
+
+**Warning:** If you reach your cancellation date before completing the archive export, the export will be cancelled and you will be unable to retrieve the rest of your archive data.
+
+If you want to export your own archive considerations the following before deciding to export an archive.
```
Comment (cyrichardson) [inline content/rackspace-email-archiving/export-an-archive-when-cancelling-your-account.md]: considerations, the following

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7fce683ca9/how-to/).

**Pull request merged**